pipelined/bevy_render2/src/color/colorspace.rs

```diff
@@ -5,6 +5,7 @@ pub trait SrgbColorSpace {
 
 // source: https://entropymine.com/imageworsener/srgbformula/
 impl SrgbColorSpace for f32 {
+    #[inline]
```

Comment:
Did all these inlines make a significant difference?

Reply:
They do. The color conversion was (and continues to be) an expensive per-sprite operation and this did produce measurable improvements.